kyuubi-server/src/main/scala/org/apache/kyuubi/server/KyuubiRestFrontendService.scala

```diff
@@ -98,19 +98,13 @@ class KyuubiRestFrontendService(override val serverable: Serverable)
     val proxyHandler = ApiRootResource.getEngineUIProxyHandler(this)
     server.addHandler(authenticationFactory.httpHandlerWrapperFactory.wrapHandler(proxyHandler))
 
-    server.addStaticHandler("org/apache/kyuubi/ui/static", "/static/")
-    server.addRedirectHandler("/", "/static/")
-    server.addRedirectHandler("/static", "/static/")
-    server.addStaticHandler("META-INF/resources/webjars/swagger-ui/4.9.1/", "/swagger-static/")
```

Comment:
it's safety to remove dep `swagger-ui`

https://github.com/apache/kyuubi/blob/5cff4fb98cdc8d8a8d9a1c289a17a17a52a787e5/pom.xml#L836-L846

Reply:
thanks for catching, removed